### PR TITLE
Add alm-examples to OLM bundle

### PR DESCRIPTION
**Pull request by rollandf**
Signed-off-by: Fred Rolland <frolland@nvidia.com>

### CODE REVIEW

File: config/samples/mellanox.com_v1alpha1_hostdevicenetwork.yaml [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+  resourceName: "hostdev"
+  ipam: |
+    {
+      "type": "whereabouts",
+      "range": "192.168.3.225/28",
+      "exclude": [
+       "192.168.3.229/30",
+       "192.168.3.236/32"
+      ]
+    }
```
Comment (adrianchiris) [inline config/samples/mellanox.com_v1alpha1_hostdevicenetwork.yaml:29]: nit: newline

File: config/samples/mellanox.com_v1alpha1_macvlannetwork.yaml [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
-  # Add fields here
-  foo: bar
+  networkNamespace: "default"
+  master: "ens2f0"
+  mode: "bridge"
+  mtu: 1500
+  ipam: |
+    {
+      "type": "whereabouts",
+      "range": "192.168.2.225/28",
```
Comment (adrianchiris) [inline config/samples/mellanox.com_v1alpha1_macvlannetwork.yaml:26]: can we change the example to be /24 ?

currently this gives you 11 addresses (with the exclude list below)

File: config/samples/mellanox.com_v1alpha1_hostdevicenetwork.yaml (hunk 1 above)
Comment (rollandf) [reply config/samples/mellanox.com_v1alpha1_hostdevicenetwork.yaml]: Done

File: config/samples/mellanox.com_v1alpha1_macvlannetwork.yaml (hunk 2 above)
Comment (rollandf) [reply config/samples/mellanox.com_v1alpha1_macvlannetwork.yaml]: Done

**Review by e0ne — APPROVED** [review]

**Pull request merged**